### PR TITLE
Ranking table

### PR DESCRIPTION
**Pull request by FRYoussef**
Changes:
   * Added a table which represents the ranking by a local metric
   * The table allows sorting and pagging
   *  The table allows to highlight a selected node

### CODE REVIEW

**Comment by Akronix** [comment]
What happened with the `wikichron/dash/main.py` file?
It looks like if the whole file changed :confused: 
The rest looks good to me.

**Comment by FRYoussef** [comment]
That's because when i tried to merge it marked the whole file as a conflict

**Comment by Akronix** [comment]
The problem was that you changed the unix line breaks to Windows line breaks in the `wikichron/dash/main.py` file. I have switched them back to unix ones and now the diff it's alright.
Remind to always have set LF for line breaks for all the files in this project.

File: wikichron/dash/networks/models/CoEditingNetwork.py [hunk 1]
```diff
@@ -237,3 +252,24 @@ def remove_non_article_data(self, df):
...
+
+
+    @classmethod
+    def get_available_metrics(cls) -> dict:
+        return cls.AVAILABLE_METRICS
+
+    
+    @classmethod
+    def get_user_info(cls) -> dict:
+        return cls.USER_INFO
```
Comment (Akronix) [inline wikichron/dash/networks/models/CoEditingNetwork.py]: Doesn't your editor have an option to always set a newline at end of file?

**Pull request merged**